_data/projects/search/releases/6.0/6.0.0.CR1.yml

```diff
@@ -0,0 +1,10 @@
+date: 2020-11-03
```

Comment:
According to the blog the release date is "2020-11-04". Probably it would be better to be consistent, aligning one of the tow.